debian/ala-namematching-service.postinst

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+#######################################################
+#
+# Add here the commands to be excuted after the package
+# has been installed
+#
+
+#Example: create some symbolic links
+#
+#chown -R ${user}:${group} /opt/${appName}-${appVersion}
+#ln -s /opt/${appName}-${appVersion} /opt/${appName}
+
+set -e
+
+. /usr/share/debconf/confmodule
+
+#DEBHELPER#
+
+case "$1" in
+    configure)
+        # We read the URL of the nameindex asked to the user in a interactive mode
+        db_get ala-namematching-service/source
+
+        # To prevent /data/lucene installed nameindexes to fill our disk:
+        #
+        # 1) we try to not download the same nameindex again and again (this is why we use wget -nc)
+        # 2) we rsync the tar over /data/lucene/namematching overwritting the previous nameindex
+        #
+
+        echo "Downloading nameindex $RET ..."
+        curl -sLC - -o "/data/lucene/namematching.tgz" "$RET"
```

Comment:
I just found that this curl command sometimes does not works as expected. Works well with truncated/partial files. 

Maybe I have to add a checksum dialog box, because other things I've tested don't work neither, like to check the size:
```
$ curl -sI https://archives.ala.org.au/archives/nameindexes/latest/namematching-20200214.tgz | awk '/Content-Length/ { print $2 }'   # this doesn't returns the size
$  curl -sI https://datos.gbif.es/others/namematching-gbif-backbone-2020-06-18.tgz | awk '/Content-Length/ { print $2 }' # this returns the size
1196260529
```

Reply:
Ok, I solved it adding an extra SHA1 check.